src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -5,6 +5,9 @@
 #include "Terminal.hpp"
 #include "unicode.hpp"
 
+#define TIL_POINT_WRAP(coord) \
+    til::point { coord.X, coord.Y }

```

Comment:
Shouldn't `til::point{ coord }` work directly? In the 1.14 branch that constructor should still exist.

Reply:
same question!